apps/src/code-studio/levels/codeStudioLevels.js

```diff
@@ -108,16 +108,22 @@ export function lockContainedLevelAnswers() {
   levelIds.forEach(levelId => getLevel(levelId).lockAnswers());
 }
 
-export function getContainedLevelResults() {
+export function getContainedLevelResult() {
```

Comment:
We had a method to get multiple results, but only ever got the first result (I think Chris was trying to build this to be more flexible). This simplifies the code to only ever support a single containedLevel/result. If we need more in the future (talking to Poorva it sounds like we don't expect that), it shouldn't be too difficult to expand for that case at that point.